xbmc/commons/ilog.h

```diff
@@ -17,12 +17,10 @@
 // ones we use in the code
 #define LOGDEBUG   0
 #define LOGINFO    1
-#define LOGNOTICE  2
-#define LOGWARNING 3
-#define LOGERROR   4
-#define LOGSEVERE  5
-#define LOGFATAL   6
-#define LOGNONE    7
+#define LOGWARNING 2
```

Comment:
There are a few add-ons that make use of the integers/magic numbers directly. Vrt.nu if I am not mistaken does it. @mediaminister @dagwieers et al

Reply:
I wonder why they do that. It's not hard to change back if we need to.

Reply:
We think it is useful for VOD add-ons to make a distinction between:
- No logging (except fatal/error)
- Log network access only (URLs)
- Log everything (caching, menu access, changes, etc)

Reply:
Ok, but why does that mean you need to use the integers directly?

Reply:
It is defined as integers, so it doesn't really matter.
Mostly to keep log statements brief.

Our log statements are templated, which means the log level needs to be the first argument.
If Python 2.7 disappears we can use f-strings and things will be easier and more concise by nature.

But it shouldn't matter except for the people working on the add-on.